frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -188,6 +200,12 @@ function appReducer(state: State, action: AppAction): State {
         ...state,
         isPollsOpen: !state.isPollsOpen,
       };
+    case 'ballotBagReplaced':
+      return {
+        ...state,
+        ballotBagLastReplacedAt:
+          state.ballotBagLastReplacedAt + BALLOT_BAG_CAPACITY,
```

Comment:
This seems a bit tricky. Yes, it's true that at the time this action is sent, the  current ballot count  equals the last replaced count plus the capacity, but that takes a bit of reasoning out. I think a slightly clearer way would be to send the current ballot count as part of the action, e.g.:

```ts
{ type: 'ballotBagReplaced', currentBallotCount: number }
 ```

Reply:
Sounds good.